src/Sylius/Behat/Context/Transform/ProductVariantContext.php

```diff
@@ -41,6 +42,22 @@ public function __construct(ProductRepositoryInterface $productRepository, Produ
     }
 
     /**
+     * @Transform :variant
+     * @Transform /^in variant "["^]+"$/
```

Comment:
`this product is available in variant "XXX"`? The first part would be resolved with lexical context.